src/frontend/src/lib/types/transaction.ts

```diff
@@ -12,3 +12,5 @@ export type Transaction = Omit<EthTransaction, 'data'> &
 export type TransactionFeeData = Pick<FeeData, 'maxFeePerGas' | 'maxPriorityFeePerGas'> & {
 	gas: BigNumber;
 };
+
+export type TransactionType = 'send' | 'receive';
```

Comment:
There is already `IcTransactionType` so this should probably be scoped to `EthTransactionType`. Or we introduce `TransactionType` equals to send and receive and then `IcTransactionType` extends it or else.